spec/models/component_validations/base_component_validation_spec.rb

```diff
@@ -68,13 +70,11 @@
         end
 
         it 'returns truthy' do
```

Comment:
Should probably change the name of the expectation 🙂
```suggestion
        it 'returns falsey' do
```